pkg/asset/internal/templates.go

```diff
@@ -913,21 +913,71 @@ spec:
     - --name=boot-etcd
     - --listen-client-urls=http://0.0.0.0:12379
     - --listen-peer-urls=http://0.0.0.0:12380
-    - --advertise-client-urls=http://$(MY_POD_IP):12379
-    - --initial-advertise-peer-urls=http://$(MY_POD_IP):12380
-    - --initial-cluster=boot-etcd=http://$(MY_POD_IP):12380
+    - --advertise-client-urls=http://{{ .BootEtcdServiceIP }}:12379
```

Comment:
The current code compiles in the template here:

https://github.com/kubernetes-incubator/bootkube/blob/master/pkg/util/etcdutil/migrate.go#L121-L145

Tracking issue is #476.